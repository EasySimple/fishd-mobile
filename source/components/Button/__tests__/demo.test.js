import webDemoTest from '../../../tests/demoTest';

webDemoTest('Button');
