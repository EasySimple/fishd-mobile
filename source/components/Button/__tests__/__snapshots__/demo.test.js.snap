// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Button demo correctly 1`] = `
<div>
  <div
    class="button-type-md"
  >
    <div
      class="label"
    >
      按钮类型
    </div>
    <div
      class="wrapper"
    >
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          主要按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-guide fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          引导按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-default fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          默认按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-danger fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          危险按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-warning fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          警告按钮
        </span>
      </a>
    </div>
  </div>
  <div
    class="button-plain-md"
  >
    <div
      class="label"
    >
      朴素按钮
    </div>
    <div
      class="wrapper"
    >
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-plain fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          朴素按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-guide fm-button-plain fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          朴素按钮
        </span>
      </a>
    </div>
  </div>
  <div
    class="button-hairline-md"
  >
    <div
      class="label"
    >
      细边框按钮
    </div>
    <div
      class="wrapper"
    >
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-plain fm-button-hairline fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          细边框按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-guide fm-button-plain fm-button-hairline fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          细边框按钮
        </span>
      </a>
    </div>
  </div>
  <div
    class="button-hairline-md"
  >
    <div
      class="label"
    >
      禁用状态
    </div>
    <div
      class="wrapper"
    >
      <a
        aria-disabled="true"
        class="fm-button fm-button-primary fm-button-normal fm-button-disabled"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          禁用状态
        </span>
      </a>
      <a
        aria-disabled="true"
        class="fm-button fm-button-guide fm-button-plain fm-button-normal fm-button-disabled"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          禁用状态
        </span>
      </a>
    </div>
  </div>
  <div
    class="button-loading-md"
  >
    <div
      class="label"
    >
      加载状态
    </div>
    <div
      class="wrapper"
    >
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-normal fm-button-loading fm-button-icon"
        role="button"
      >
        <i
          class="icon iconfont fm-icon fm-icon-loading"
        />
        <span
          class="fm-button-text"
        >
          加载状态
        </span>
      </a>
    </div>
  </div>
  <div
    class="button-loading-md"
  >
    <div
      class="label"
    >
      图标按钮
    </div>
    <div
      class="wrapper"
    >
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-normal fm-button-icon"
        role="button"
      >
        <i
          class="icon iconfont fm-icon fm-icon-plus-x1x"
        />
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-guide fm-button-normal fm-button-icon"
        role="button"
      >
        <i
          class="icon iconfont fm-icon fm-icon-plus-x1x"
        />
        <span
          class="fm-button-text"
        >
          按 钮
        </span>
      </a>
    </div>
  </div>
  <div
    class="button-size-md"
  >
    <div
      class="label"
    >
      按钮形状
    </div>
    <div
      class="wrapper"
    >
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-normal fm-button-square"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          方形按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-normal fm-button-round"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          圆形按钮
        </span>
      </a>
    </div>
  </div>
  <div
    class="button-size-md"
  >
    <div
      class="label"
    >
      按钮尺寸
    </div>
    <div
      class="wrapper"
    >
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-large"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          大号按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          普通按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-small"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          小型按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-primary fm-button-mini"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          迷你按钮
        </span>
      </a>
    </div>
  </div>
  <div
    class="button-color-md"
  >
    <div
      class="label"
    >
      块级元素
    </div>
    <div
      class="wrapper"
    >
      <a
        aria-disabled="false"
        class="fm-button fm-button-guide fm-button-normal fm-button-block"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          块级元素
        </span>
      </a>
    </div>
  </div>
  <div
    class="button-color-md"
  >
    <div
      class="label"
    >
      自定义颜色
    </div>
    <div
      class="wrapper"
    >
      <a
        aria-disabled="false"
        class="fm-button fm-button-default fm-button-normal"
        role="button"
        style="color:#fff;background:#6850FF;border-color:#fff"
      >
        <span
          class="fm-button-text"
        >
          单色按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-default fm-button-plain fm-button-normal"
        role="button"
        style="color:#6850FF;background:#fff;border-color:#6850FF"
      >
        <span
          class="fm-button-text"
        >
          幽灵按钮
        </span>
      </a>
      <a
        aria-disabled="false"
        class="fm-button fm-button-default fm-button-normal"
        role="button"
        style="color:#fff;background:linear-gradient(269deg, #A783FF 0%, #7921FF 100%);border-color:#fff"
      >
        <span
          class="fm-button-text"
        >
          渐变色按钮
        </span>
      </a>
    </div>
  </div>
</div>
`;
